src/main/java/teammates/ui/controller/InstructorEditInstructorFeedbackPageAction.java

```diff
@@ -34,15 +34,9 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                                 
         String moderatedQuestionId = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_MODERATED_QUESTION_ID);
 
-        Assumption.assertNotNull(String.format(Const.StatusMessages.NULL_POST_PARAMETER_MESSAGE,
-                                               Const.ParamsNames.COURSE_ID),
-                                 courseId);
-        Assumption.assertNotNull(String.format(Const.StatusMessages.NULL_POST_PARAMETER_MESSAGE,
-                                               Const.ParamsNames.FEEDBACK_SESSION_NAME),
-                                 feedbackSessionName);
-        Assumption.assertNotNull(String.format(Const.StatusMessages.NULL_POST_PARAMETER_MESSAGE,
-                                               Const.ParamsNames.FEEDBACK_SESSION_MODERATED_PERSON),
-                                 instructorUnderModerationEmail);
+        Assumption.assertNotNull(Const.StatusMessages.NULL_POST_PARAMETER_MESSAGE, courseId);
```

Comment:
Well the previous one doesn't help either. There's _no_ `%s` in the said variable to be replaced with the parameter name.